hummingbot/cli/ui/completer.py

```diff
@@ -1,14 +1,10 @@
-import asyncio
-from typing import (
-    List,
-    Dict,
-)
+import re
+from typing import List
 from prompt_toolkit.completion import Completer, WordCompleter, PathCompleter, CompleteEvent
 from prompt_toolkit.document import Document
 
-from hummingbot.cli.settings import EXCHANGES, STRATEGIES
+from hummingbot.cli.settings import EXCHANGES, STRATEGIES, CONF_FILE_PATH, symbol_fetcher
```

Comment:
Please use the new multi-line import style.

Reply:
fixed